<a href="https://colab.research.google.com/github/sharclark/Project1_Dojo/blob/main/Data_options_Project_2_Shar.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [4]:
import pandas as pd
df_wine = pd.read_csv('/content/wines_SPA.csv')
df_wine.head()

winery           wine  year  rating  num_reviews country  \
0  Teso La Monja          Tinto  2013     4.9           58  Espana   
1         Artadi  Vina El Pison  2018     4.9           31  Espana   
2   Vega Sicilia          Unico  2009     4.8         1793  Espana   
3   Vega Sicilia          Unico  1999     4.8         1705  Espana   
4   Vega Sicilia          Unico  1996     4.8         1309  Espana   

             region   price                  type  body  acidity  
0              Toro  995.00              Toro Red   5.0      3.0  
1    Vino de Espana  313.50           Tempranillo   4.0      2.0  
2  Ribera del Duero  324.95  Ribera Del Duero Red   5.0      3.0  
3  Ribera del Duero  692.96  Ribera Del Duero Red   5.0      3.0  
4  Ribera del Duero  778.06  Ribera Del Duero Red   5.0      3.0

In [6]:
df_wine.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 7500 entries, 0 to 7499
Data columns (total 11 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   winery       7500 non-null   object 
 1   wine         7500 non-null   object 
 2   year         7498 non-null   object 
 3   rating       7500 non-null   float64
 4   num_reviews  7500 non-null   int64  
 5   country      7500 non-null   object 
 6   region       7500 non-null   object 
 7   price        7500 non-null   float64
 8   type         6955 non-null   object 
 9   body         6331 non-null   float64
 10  acidity      6331 non-null   float64
dtypes: float64(4), int64(1), object(6)
memory usage: 644.7+ KB


In [13]:
df_ecom_shipping = pd.read_csv('/content/Train.csv')
df_ecom_shipping.head()

ID Warehouse_block Mode_of_Shipment  Customer_care_calls  Customer_rating  \
0   1               D           Flight                    4                2   
1   2               F           Flight                    4                5   
2   3               A           Flight                    2                2   
3   4               B           Flight                    3                3   
4   5               C           Flight                    2                2   

   Cost_of_the_Product  Prior_purchases Product_importance Gender  \
0                  177                3                low      F   
1                  216                2                low      M   
2                  183                4                low      M   
3                  176                4             medium      M   
4                  184                3             medium      F   

   Discount_offered  Weight_in_gms  Reached.on.Time_Y.N  
0                44           1233                    1  
1                59           3088                    1  
2                48           3374                    1  
3                10           1177                    1  
4                46           2484                    1

In [15]:
df_ecom_shipping.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 10999 entries, 0 to 10998
Data columns (total 12 columns):
 #   Column               Non-Null Count  Dtype 
---  ------               --------------  ----- 
 0   ID                   10999 non-null  int64 
 1   Warehouse_block      10999 non-null  object
 2   Mode_of_Shipment     10999 non-null  object
 3   Customer_care_calls  10999 non-null  int64 
 4   Customer_rating      10999 non-null  int64 
 5   Cost_of_the_Product  10999 non-null  int64 
 6   Prior_purchases      10999 non-null  int64 
 7   Product_importance   10999 non-null  object
 8   Gender               10999 non-null  object
 9   Discount_offered     10999 non-null  int64 
 10  Weight_in_gms        10999 non-null  int64 
 11  Reached.on.Time_Y.N  10999 non-null  int64 
dtypes: int64(8), object(4)
memory usage: 1.0+ MB


# Shipping Data - First Choice


1. Source of data
  - Kaggle

2. Brief description of data
  - Ecom company shipping data

3. What is the target?
  - reached on time column

4. What does one row represent? 
  - an order and whether or not it shipped on time and other features

5. Is this a classification or regression problem?
  - classification

6. How many features does the data have?
  - 12, including the target

7. How many rows are in the dataset?
  - 10,999
8. What, if any, challenges do you foresee in cleaning, exploring, or modeling this dataset?
  - there are no missing data, so there may have already been some imputation, so the data could be compromised. There is a possibility that there isnt a correlation between on time and other features. Shipping always has external factors that affect if things are on time or not. 

# Wine Data

1. Source of data 
  - Kaggle (web scraping)

2. Brief description of data 
  - 7500 different types of wine from Spain

3. What is the target? 
  - price would be obvious, but i want to try to predict the score

4. What does one row represent? 
  - a wine

5. Is this a classification or regression problem?
  - regression

6. How many features does the data have?
  - 11 different features

7. How many rows are in the dataset?
  - 7500

8. What, if any, challenges do you foresee in cleaning, exploring, or modeling this dataset?
  - I have a background in wine (operations) and i doubt this model, or any model, will be able to predict prices. BUT I think this data set would work well for predicting the rating of the wines. 